webapp/ansible/roles/service-vm/tasks/github.yml

```diff
@@ -0,0 +1,3 @@
+---
+  - name: Download Django code from Github.
+    git: repo=https://github.com/gouzouni625/okeanos-LoD.git dest=/root/okeanos-LoD
```

Comment:
I am guessing you are using your fork here because this (#35) is not already merged in, right?